package.json

```diff
@@ -22,7 +22,7 @@
     "colors-tmpl": "~0.1.0",
     "core-util-is": "~1.0.1",
     "cpr": "~0.3.2",
-    "getos": "~1.0.1",
+    "getos": "^2.3.0",
```

Comment:
because `^` is an absurd range for semver, so set your default to `~` when you do an `npm install foo --save`

just a suggestion of course, personal preference